src/configure.rs

```diff
@@ -442,7 +442,6 @@ async fn configure(ctx: &Context, param: &EnteredLoginParam) -> Result<Configure
         ctx.set_config(Config::MvboxMove, Some("0")).await?;
         ctx.set_config(Config::OnlyFetchMvbox, None).await?;
         ctx.set_config(Config::ShowEmails, None).await?;
-        ctx.set_config(Config::E2eeEnabled, Some("1")).await?;
```

Comment:
So if encryption preference is still sent in the Autocrypt header, can't this lead to other Autocrypt-capable MUA or old Delta Chat sending messages unencrypted?

Reply:
It can be disabled by the user, but the correct way to fix this is to remove the setting from UIs. Indeed, no need to reenable it here